src/admin/gutenberg/sesamy-post-editor/src/sesamyPostEditor.js

```diff
@@ -57,15 +58,16 @@ const SesamyPostEditor = () => {
 
 	wp.data
 		.dispatch("core/edit-post")
-		.removeEditorPanel("taxonomy-panel-sesamy_tags"); //Hide sesamy_tags taxonomy panel from sidebar
-	// wp.data.dispatch( 'core/editor').removeEditorPanel( 'taxonomy-panel-sesamy_tags' ) ; //Hide sesamy_tags taxonomy panel from sidebar
+		.removeEditorPanel("taxonomy-panel-sesamy_tags");
 
 	const sesamyTiersTaxonomy = useSelect((select) => {
-		return select(coreStore).getEntityRecords("taxonomy", "sesamy_passes");
+		return (
+			select(coreStore).getEntityRecords("taxonomy", "sesamy_passes") || []

```

Comment:
Added a fallback in case the sesamy_passes are undefined, which is an error we seen before.